src/GiftedChat.tsx

```diff
@@ -66,7 +66,7 @@ import { useSafeAreaInsets } from 'react-native-safe-area-context'
 
 dayjs.extend(localizedFormat)
 
-export interface GiftedChatProps<TMessage extends IMessage = IMessage> {
+export interface GiftedChatProps<TMessage extends IMessage = IMessage> extends Partial<Omit<MessageContainer<TMessage>, 'scrollToBottom'>> {
```

Comment:
had to omit `scrollToBottom` because of the name collision.. don't want to add a breaking change by renaming this